docs/content/en/integration/openid-connect/seafile/index.md

```diff
@@ -17,8 +17,10 @@ community: true
 
 * [Authelia]
   * [v4.36.9](https://github.com/authelia/authelia/releases/tag/v4.36.9)
+  * [v4.37.5](https://github.com/authelia/authelia/releases/tag/v4.37.5)
```

Comment:
```suggestion
  * [v4.36.9](https://github.com/authelia/authelia/releases/tag/v4.36.9)
  * [v4.38.0](https://github.com/authelia/authelia/releases/tag/v4.38.0)
```

As above, I'll figure out how to reflect the adjustments for users on the older versions later.